processor/remotetapprocessor/README.md

```diff
@@ -24,8 +24,8 @@ any open WebSockets is rate limited by an adjustable amount.
 
 The WebSocket processor has two configurable fields: `port` and `limit`:
 
-- `port`: The port on which the WebSocket processor listens. Optional. Defaults
-  to `12001`.
+- `endpoint`: The endpoint on which the WebSocket processor listens. Optional. Defaults
+  to `0.0.0.0:12001`.
```

Comment:
was this change related to the bug fix?

Reply:
Document error. Has nothing to do with bug fixes.

Reply:
It can be a separate PR if needed.

Reply:
Okay, a new PR  for `README.md`